--- ./src/plugins/alsa/Makefile.am.orig	2016-12-17 18:29:25.013082712 +0200
+++ ./src/plugins/alsa/Makefile.am	2016-12-17 18:30:16.370147896 +0200
@@ -20,7 +20,7 @@ libdir = $(plugindir)
 lib_LTLIBRARIES = $(alsaltlibs)
 
-libalsa_la_LDFLAGS = @PLUGIN_LDFLAGS@
+libalsa_la_LDFLAGS = @PLUGIN_LDFLAGS@ -L$(top_srcdir)/src/.libs/
-libalsa_la_LIBADD = @ALSA_LIBS@
+libalsa_la_LIBADD = $(top_srcdir)/src/libao.la @ALSA_LIBS@
 libalsa_la_SOURCES = $(alsasources)
 
 EXTRA_DIST = ao_alsa.c

